plugins/cryptolib/cryptolib.cpp

```diff
@@ -436,7 +482,19 @@ class CDSMCache
         else
         {
             if (!isEmptyString(pubKeyFS) || !isEmptyString(privKeyFS))
-                ret = createDigitalSignatureManagerInstanceFromFiles(pubKeyFS, privKeyFS, passphrase);
+            {
+                bool isPublic = isEmptyString(privKeyFS);
+                const char * fs = isPublic ? pubKeyFS : privKeyFS;;
+
+                if (isLFN)
+                {
+                    StringBuffer sb;
+                    loadLFS(fs, user, sb);//read key file into StringBuffer
+                    ret = createDigitalSignatureManagerInstanceFromKeys(isPublic ? sb.str() : nullptr, isPublic ? nullptr : sb.str(), passphrase);
+                }
+                else
```

Comment:
could do with comment saying only to support legacy direct files (unless we can remove completely).